Mathlib/Analysis/Calculus/DifferentialForm/VectorField.lean

```diff
@@ -0,0 +1,196 @@
+/-
+Copyright (c) 2025 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury G. Kudryashov
+-/
+module
+
+public import Mathlib.Analysis.Calculus.DifferentialForm.Basic
+public import Mathlib.Analysis.Calculus.FDeriv.ContinuousAlternatingMap
+public import Mathlib.Analysis.Calculus.VectorField
+
+/-!
+# Theorems about evaluation of differential forms on vector fields
+
+In this file we prove the following formula and its corollaries.
+If `ω` is a differentiable `k`-form and `V i` are `k + 1` differentiable vector fields, then
+
+$$
+  dω(V_0(x), \dots, V_n(x)) =
+    \left(\sum_{i=0}^k (-1)^i • D_x(ω(x; V_0(x), \dots, V_k(x))) +
+      \sum_{0 \le i < j\le k} (-1)^{i + j}
+        ω(x; [V_i, V_j](x), V_0(x), …, \widehat{V_i(x)}, …, \widehat{V_j(x)}, …, V_k(x)),
+$$
+where $$[V_i, V_j]$$ is the commutator of the vector fields $$V_i$$ and $$V_j$$.
+As usual, $$\widehat{V_i(x)}$$ means that this item is removed from the sequence.
+
+There is no convenient way to write the second term in Lean for `k = 0`,
+so we only state this theorem for `k = n + 1`,
+see `extDerivWithin_apply_vectorField` and `extDeriv_apply_vectorField`.
+
+In this case, we write the second term as a sum over `i j : Fin (n + 1)`, `i ≤ j`,
+where the indexes `(i, j)` in our sum currespond to `(i, j + 1)`
+(formally, `(Fin.castSucc i, Fin.succ j)`) in the formula above.
+For this reason, we have `-` before the sum in our formal statement.
+-/
```

Comment:
Can you please write something about which 'enduser' versions you have and how they differ?

Reply:
UPD: we can add specializations of these theorems to `n = 1, 2` once we need them.